message_definitions/v1.0/messages.xml

```diff
@@ -706,25 +592,8 @@
       <field name="index_list" type="uint8[]"/>
     </message>
 
-    <message name="CROSS_TRACK_ERROR" id="80">
-      <field name="cross_track_error" type="float" unit="m"/>
-      <field name="cte_int" type="float"/>
-    </message>
-
-    <message name="GENERIC_COM" id="81">
```

Comment:
Even if not really used at the moment, it is still part of the ground side and it corresponds to a message sent over satcom link.